docs/source/algorithms/SANSSolidAngle-v1.rst

```diff
@@ -0,0 +1,29 @@
+
+.. algorithm::
+
+.. summary::
+
+.. relatedalgorithms::
+
+.. properties::
+
+Description
+-----------
+
+Performs a solid angle calculation on all detector pixels.
+
+Depending on the option selected, the solid angle is calculated as follows:
```

Comment:
I suggest clearly explaining the difference to the other existing algorithms, as you outlined in the PR message.

Also, should existing solid-angle algorithms reference this in their "see also"?

Reply:
This is an internal document. I'm not sure if we can share this. There's a lot of "secrecy" at the ORNL.... The idea is to get rid of the workflow algorithms with time (the old sans solid angle correction).

The main computation for SANS is here:
```c++
	  if (type == "Normal") {
	    return [&spectrumInfo](int64_t histogramIndex) {
	      double cosTheta = std::cos(spectrumInfo.twoTheta(histogramIndex));
	      return cosTheta * cosTheta * cosTheta;
	    };
	  } else if (type == "Tube") {
	    return [&spectrumInfo](int64_t histogramIndex) {
	      double cosTheta = std::cos(spectrumInfo.twoTheta(histogramIndex));
	      double cosAlpha = std::cos(getYTubeAngle(spectrumInfo, histogramIndex));
	      return cosTheta * cosTheta * cosAlpha;
	    };
	  } else if (type == "Wing") {
	    return [&spectrumInfo](int64_t histogramIndex) {
	      double cosAlpha = std::cos(getYTubeAngle(spectrumInfo, histogramIndex));
	      return cosAlpha * cosAlpha * cosAlpha;
	    };
```

Reply:
We're refactoring the SANS code in such a way that `SANSSolidAngle` will only compute the solid angle. A simple divide will follow. The existing, which will eventually disappear, both calculates and applies. So those two algorithms should not refer to each other (no "see also"), and should live side-by-side for the time being until we phase out the existing one.

Reply:
Should you deprecate the older then? Do you know if it is used outside ORNL? Have you considered using `SolidAngle` algorithm or is it too slow?

Reply:
I think there is a bit of a confusion, when you say existing, you refer to `SANSSolidAngleCorrection`. But I think you do have to link to `SolidAngle` and back and make it clear what's the difference.